tests/test_regex_identifier.py

```diff
@@ -475,6 +478,47 @@ def test_arn4():
     res = r.check(["arn:aws:s3:::my_corporate_bucket/Development/*"])
     assert "ARN" in str(res)
 
+
+def test_unix_timestamp1():
```

Comment:
Pls remove 1 here and it will be good to go

Reply:
adjusted.